pkgs/data/fonts/culmus/default.nix

```diff
@@ -3,17 +3,24 @@
 let
   version = "0.133";
 in fetchzip {
-  name = "culmus-${version}";
+  name = "culmus-${version}-1";
```

Comment:
This PR is a bugfix. No need to append the version.

Reply:
sorry, changed it so it will rebuild locally, need to be squashed ?

Reply:
I can squash everything for the merge in the end.

Reply:
Done